deploy/examples/cluster-multus-test.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
 spec:
   dataDirHostPath: /var/lib/rook
   cephVersion:
-    image: quay.io/ceph/ceph:v17
+    image: quay.io/ceph/ceph:v18
```

Comment:
It might be worth separating out the v18 image changes to a separate commit to make it obvious in the commit history that we updated the default to Reef.